### PR TITLE
Better `null` handling for `getOption`.

### PR DESCRIPTION
**Pull request by ddebin**

### CODE REVIEW

File: tests/acceptance/acceptance/ConsoleOption.feature [hunk 1]
```diff
@@ -158,15 +164,26 @@ Feature: ConsoleOption
         {
           $this->addOption('optional_string1');
           $this->addOption('optional_string2', null, InputOption::VALUE_OPTIONAL);
+          $this->addOption('optional_string3', null, InputOption::VALUE_REQUIRED);
+          $this->addOption('required_array', null, InputOption::VALUE_OPTIONAL | InputOption::VALUE_IS_ARRAY);
```
[the remark this answers is not in the document]
Comment (ddebin) [reply tests/acceptance/acceptance/ConsoleOption.feature]: OK. Done.

File: tests/acceptance/acceptance/ConsoleOption.feature [hunk 2]
```diff
@@ -158,15 +164,26 @@ Feature: ConsoleOption
         {
           $this->addOption('optional_string1');
           $this->addOption('optional_string2', null, InputOption::VALUE_OPTIONAL);
+          $this->addOption('optional_string3', null, InputOption::VALUE_REQUIRED);
```
[the remark this answers is not in the document]
Comment (ddebin) [reply tests/acceptance/acceptance/ConsoleOption.feature]: OK. Done.

File: tests/acceptance/acceptance/ConsoleOption.feature [hunk 3]
```diff
@@ -50,7 +50,7 @@ Feature: ConsoleOption
         public function configure(): void
         {
           $this->addOption('required_array', null, InputOption::VALUE_REQUIRED | InputOption::VALUE_IS_ARRAY);
-          $this->addOption('required_string', null, InputOption::VALUE_REQUIRED);
+          $this->addOption('required_string', null, InputOption::VALUE_REQUIRED, '', 'default');
```
[the remark this answers is not in the document]
Comment (ddebin) [reply tests/acceptance/acceptance/ConsoleOption.feature]: I've added a separate case.

**Comment by seferov** [comment]
@ddebin thank you!

**Pull request merged**